### ISSUE TITLE
Alternate attendance options

### ISSUE THREAD

**Issue by alanzhu0** [tjcsl/ion#1612]
Ion should support more methods for taking attendance. Two main Ion-supported ways are currently used:
- Teacher calls and marks students 
- Students check off on printed attendance sheet

Other non-official ways include:
- Signing in by filling out a Google Form (especially for large activities)

---

Ideas for new attendance methods:
- Scanning a QR code/link -> takes students to a page that automatically marks them present
  - Keep track of time response is recorded (should be within block)
- Students input a code (that the teacher displays) into an attendance box on Ion
- ...more ideas welcome

**Comment by alanzhu0** [comment]
Yep, that too. Thanks for adding that in. They are actually using that feature to process the Google Form attendance spreadsheet too.

**Comment by TigerCodeHub** [comment]
Can you please assign this issue to me? Thanks!

Tiger Deng
TJHSST Class of 2028

**Comment by NotFish232** [comment]
Sure thing!
